### PR TITLE
Note that not all MOP.RR.7 or MOP.R.28 encodings are used for Zicfiss

### PR DESCRIPTION
**Pull request by ved-rivos**

### CODE REVIEW

File: src/cfi_backward.adoc [hunk 1]
```diff
@@ -98,8 +98,9 @@ The Zicfiss extension introduces the following instructions:
 * Perform an atomic swap from a shadow stack location (See <<SSAMOSWAP>>)
 ** `SSAMOSWAP.W` and `SSAMOSWAP.D`
 
-When a `MOP.RR.7` or `MOP.R.28` encoding is not utilized by the Zicfiss
-extension, the instruction adheres to its Zimop-defined behavior, unless it is
+Zicfiss does not use all encodings of `MOP.RR.7` or `MOP.R.28`. When a
+`MOP.RR.7` or `MOP.R.28` encoding is not utilized by the Zicfiss extension, the
+corresponding instruction adheres to its Zimop-defined behavior, unless it is
 employed by another extension. In such cases, the instruction follows the
 behavior specified by that other extension.
```
Comment (aswaterman) [inline src/cfi_backward.adoc:105]: I suggest shortening `unless it is employed by another extension. In such cases, the instruction follows the behavior specified by that other extension.` to `unless redefined by another extension.`  This matches the language in the Zimop spec and avoids making it seem like this is some kind of special case.

**Review by aswaterman — APPROVED** [review]
LGTM modulo my one comment

File: src/cfi_backward.adoc (hunk 1 above)
Comment (ved-rivos) [reply src/cfi_backward.adoc]: @aswaterman updated. Thanks!

**Review by kasanovic — COMMENTED** [review]
I would change "utilized" to "used" to more clearly match the "does not use" in first sentences.  It's also shorter.

**Comment by ved-rivos** [comment]
@kasanovic updated as suggested.

**Pull request merged**